costs/es_request_constructor.go

```diff
@@ -0,0 +1,254 @@
+//   Copyright 2017 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package costs
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"gopkg.in/olivere/elastic.v5"
+)
+
+// paramNameToFuncPtr : map of string keys and functions pointer as values. For each possible param
```

Comment:
This is not the correct GoDoc syntax. Your comment should start with _a full sentence_ whose first word is the identifier. Think that godoc will show just the first sentence when it does not have room for the full description, so it better be useful on its own and look like an actual sentence. For example :
```go
// paramNameToFuncPtr maps parameter names to the functions building the corresponding aggregation.
// …
```